compare.py

```diff
@@ -0,0 +1,76 @@
+#! /usr/bin/python
+#
+# Compares the speed of two different compilers on one or more benchmarks in
+# the suite.
+
+from __future__ import division, print_function
+
+import os
+import subprocess
+import sys
+import time
+
+def run_test(dir, rustc):
+    os.chdir(dir)
+
+    # First clean and rebuild from scratch. This downloads any necessary code
+    # and builds preliminary crates.
+    #
+    # We use call() instead of check_call() for `make clean` because it can
+    # fail reasonably, e.g. if we try to delete a Cargo.lock file thta isn't
+    # present.
+    subprocess.call('make clean > /dev/null 2>&1', shell=True)
+    make_env = os.environ.copy()
+    make_env['RUSTC'] = rustc
+    subprocess.check_call('make > /dev/null 2>&1', env=make_env, shell=True)
+
+    # Measure compilation speed of the crate of interest three times.
+    times = []
+    for i in range(0, 3):
+        subprocess.check_call('make touch > /dev/null 2>&1', shell=True)
+        t1 = time.time()
+        subprocess.check_call('make > /dev/null 2>&1', env=make_env, shell=True)
+        t2 = time.time()
```

Comment:
Agreed that finer-grained timing is a good idea, but can definitely be done later.

Rewriting these bash and python scripts in Rust or Python has been a goal of mine for a while, but since they "worked" I wasn't inclined to change anything, especially since I'm not 100% clear on what the current situation is as to what is used and what isn't.